ecosystem/python/sdk/aptos_sdk/async_client.py

```diff
@@ -259,26 +474,10 @@ async def simulate_transaction(
     ) -> Dict[str, Any]:
         # Note that simulated transactions are not signed and have all 0 signatures!
         authenticator = sender.sign_simulated_transaction(transaction)
-        signed_transaction = SignedTransaction(transaction, authenticator)
-
-        headers = {"Content-Type": "application/x.aptos.signed_transaction+bcs"}
-        params = {}
-        if estimate_gas_usage:
-            params = {
-                "estimate_gas_unit_price": "true",
-                "estimate_max_gas_amount": "true",
-            }
-
-        response = await self.client.post(
-            f"{self.base_url}/transactions/simulate",
-            params=params,
-            headers=headers,
-            content=signed_transaction.bytes(),
+        return await self.simulate_bcs_transaction(
+            signed_transaction=SignedTransaction(transaction, authenticator),
+            estimate_gas_usage=estimate_gas_usage,
```

Comment:
I don't think `estimate_gas_usage` is used in `submit_bcs_transaction`?  Might have to port this over from the old code here.

Reply:
No it's not. `estimate_gas_usage` is only used in `simulate_transaction` and `simulate_bcs_transaction`, not `submit_transaction` and `submit_bcs_transaction`. Is this an issue?

Reply:
I read wrong, read it as submit instead of simulate.  My mistake